src/main/java/mate/academy/spring/validation/RepeatPasswordValidator.java

```diff
@@ -0,0 +1,18 @@
+package mate.academy.spring.validation;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import mate.academy.spring.dto.request.UserRequestDto;
+
+public class RepeatPasswordValidator implements
+        ConstraintValidator<RepeatPassword, UserRequestDto> {
+
+    @Override
+    public boolean isValid(UserRequestDto userRequestDto,
+                           ConstraintValidatorContext constraintValidatorContext) {
+        if (userRequestDto.getRepeatPassword() == null) {
+            return false;
+        }
+        return userRequestDto.getPassword().equals(userRequestDto.getRepeatPassword());
```

Comment:
you should check getPassword() on null, not repeat password because potentially you can get NPE + rewrite your logic, so you won't need if